pennylane/interfaces/execution.py

```diff
@@ -312,6 +312,7 @@ def wrapper(tapes: Sequence[QuantumTape], **kwargs):
 def execute(
     tapes: Sequence[QuantumTape],
     device: device_type,
+    transform_program=None,

```

Comment:
This will also allow us to converge the PL python and catalyst execution pipelines -- with the goal being that we have a single `program` representation captured by the frontend, that both pipelines can execute.